### PR TITLE
New pkg request: GN

### PR DESCRIPTION
**Pull request by dengbangjie**
# GN

## Description  
GN is a meta-build system that generates build files for Ninja. It is widely used by repos related to Google, such as chromium, qcef , etc.

## Benefits
Make it easily for fedora users to build qcef, pdfium, etc.

### CODE REVIEW

File: rpms/gn/gn.spec [hunk 1]
```diff
@@ -0,0 +1,58 @@
+%global debug_package %{nil}
+
+%global commit 78db9de5aed44f03763176ac6067fb1a73798684
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:           gn
+Version:        1559
```
Comment (sztsian) [inline rpms/gn/gn.spec]: I don't see any releases for this project, so I feel you should use the way for [upstream has never choose a version](https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_has_never_chosen_a_version). In this case, I suggest you to use 
```
Version:        0
Release:        1.git%{shortcommit}%{?dist}
```

Or similar things like
```
Version:        0
Release:        1.1559git%{shortcommit}%{?dist}
```

**Review by sztsian — CHANGES_REQUESTED** [review]

File: rpms/gn/gn.spec (hunk 1 above)
Comment (sztsian) [reply rpms/gn/gn.spec]: Adding @cheese to see which is a better version.

**Comment by dengbangjie** [comment]
I correct to `Version: 0` and `1.1559git%{shortcommit}%{?dist}` .

**Comment by sztsian** [comment]
Looks good now. I'm merging for the moment and will see if there are build failures tomorrow.

**Pull request merged**